lib/certificate.js

```diff
@@ -144,17 +144,17 @@ class TrustStore {
    */
   async addRecord (sha1, tset, subj, data) {
     if (await this.hasRecords(subj)) {
-      return await this.execSQLiteQuery(`UPDATE tsettings SET sha1=x'?', tset='?', data=x'?' WHERE subj='?'`, sha1, tset, data, subj);
+      return await execSQLiteQuery(await this.getDB(), `UPDATE tsettings SET sha1=x'?', tset='?', data=x'?' WHERE subj='?'`, sha1, tset, data, subj);
```

Comment:
It would be cleaner to pull the `await this.getDB()` out to before the conditional, since both cases use it.